lib/arena.rb

```diff
@@ -1,2 +1,39 @@
 class Arena
+  attr_accessor :name, :gladiators
```

Comment:
could use `attr_reader ` for gladiators to restrict anyone from adding gladiators except through your `add_gladiator` method